src/components/Landing.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Landing = () => (
```

Comment:
Hey @dannylwe, well done with the feedback.
Please change this file/component name to something like Home.js, 
Landing is a bit misleading